src/mixin-stache-view.js

```diff
@@ -25,10 +27,19 @@ module.exports = function mixinStacheView(Base = HTMLElement) {
 
 				this.constructor[rendererSymbol] = renderer;
 			}
-
-			const frag = renderer(this, renderOptions, parentNodeList);
+			this._nodeList = nodeLists.register([], function(){}, parentNodeList || true, false);
```

Comment:
create a nodeList for the rendering.  this nodeList allows us to immediately teardown and bindings that the view has done.